code/search/SearchVariantSubsites.php

```diff
@@ -83,9 +83,16 @@ public function extractManipulationWriteState(&$writes)
                 continue;
             }
 
+            if (isset($write['fields']['SiteTree:SubsiteID'])) {
```

Comment:
Files in subsite 0 should be global, but I suppose that means they should still only be indexed once.... So... Yes?